content/engineering/cloud/security/security-onboarding.md

```diff
@@ -0,0 +1,36 @@
+## Security**🧑‍💻**
+
+**Setup**
```

Comment:
This looks like it should be a level-2 heading:

```
## Setup
```

This makes it appear in the table of contents on the right and improves accessibility.

Reply:
Got it will modify this and the next one.